### ISSUE TITLE
Sort task in build explorer

### ISSUE THREAD

**Issue by jabby** [angelozerr/jsbuild-eclipse#8]
As an enhancement, I propose to add the ability to sort tasks by category and name.  Maybe it should be a default sort.

I use the build explorer for grunt tasks. 

For grunt files, there will be 2 categories : 
- task registered by `grunt.registerTask`
- task declared in init config (grunt.initConfig)

Both categories should be sort by name.

What do you think about this ehancement proposal ?

**Comment by angelozerr** [comment]
Yes it's a good idea, just find time to implement it.

**Comment by jabby** [comment]
For information, in External Tools Configurations, a user can sort targets by name. When you have  a lot of targets, it's quite useful.

**Comment by angelozerr** [comment]
Yes I didn't remeber that I had done that:) JSBuild Eclipse is a copy/paste of Ant plugin. Can we close this issue or do we need do something?

**Comment by jabby** [comment]
In fact there is a sort by name in External Tools. But in my use case, I launch each task by double click in the build explorer. 
My enhancement proposal is for this view not for External Tools.
